tailwind.config.js

```diff
@@ -1,11 +0,0 @@
-module.exports = {
-  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
-  darkMode: false, // or 'media' or 'class'
-  theme: {
-    extend: {},
-  },
-  variants: {
-    extend: {},
-  },
-  plugins: [],
-}
```

Comment:
Ditto: I don't know how this deletion ended up in this PR, neither how to "undo" it since it does not come from an actual commit — n.b. we'll have to revert this after merging